cineast-core/src/main/java/org/vitrivr/cineast/core/features/VisualTextCoEmbedding.java

```diff
@@ -203,6 +223,77 @@ private void initializeVisualEmbedding() {
     }
   }
 
+  private float[] embedVideo(List<BufferedImage> frames) {
```

Comment:
I don't think it's ideal to use the BufferedImage here, since that circumvents the frame caching mechanism. You could just as well use the MultiImage and access the buffered one once you need it, without having to load all of them first.